projects/storefrontlib/src/cms-components/checkout/services/checkout-config.service.ts

```diff
@@ -55,6 +61,51 @@ export class CheckoutConfigService {
     return stepIndex >= 0 ? stepIndex : null;
   }
 
+  protected sortDeliveryModes(deliveryMode1, deliveryMode2) {
```

Comment:
I would change the name of this function. It doesn't sort but is comparing deliveryModes.